data/Dhruv-Sood.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "Dhruv Sood",
+    "bio": "18 | Viewing the world from a different angle | Your friendly neighborhood Geek | Tech enthusiast 👨🏻‍💻",
+    "links": [
+        {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/dhruv_sood_",
+            "icon": "FaTwitter"
+        },
+        {
+            "name": "Checkout my Youtube",
+            "url": "https://www.youtube.com/@SoodDhruv",
+            "icon": "FaYoutube"
+        }
+    ],
+    "socials": [
+        {
+            "icon": "FaTwitter",
+            "url": "https://twitter.com/dhruv_sood_"
+        },
+        {
+            "icon": "FaGithub",
+            "url": "https://github.com/Dhruv-Sood"
+        },
+        {
+            "icon": "FaYoutube",
+            "url": "https://www.youtube.com/@SoodDhruv"
+        }
+    ],
+    "tags": [
```

Comment:
You should add `tags` above just after the `bio` section.

Reply:
In json the order of the object properties does not matter

Reply:
okay, I saw the template that's why I thought that it is not right. Thanks for helping me with it Eddie.